capstone/src/main/java/com/google/sps/servlets/student/StudentServlet.java

```diff
@@ -229,10 +230,7 @@ private static void addOrRemoveItemToEntity(
       String property,
       Boolean addItem) {
     // Create empty List if property does not exist yet
-    List<String> generalList = new ArrayList<String>();
-    if (entity.getProperty(property) != null) {
-      generalList = ((ArrayList<String>) entity.getProperty(property));
-    }
+    List<String> generalList = new ArrayList(ServletUtil.getPropertyList(entity, property));
```

Comment:
Are you still supposed to have the <String> in the constructor? Or is that just with an old version of Java or something

Reply:
Ah, you're right. Just added the String part as well, will push the change in the next commit. Thanks!